collision.go

```diff
@@ -4,6 +4,7 @@ import (
 	"log"
 
 	"engo.io/ecs"
+	"github.com/luxengine/math"
```

Comment:
# Fixed

Simply `go get -u github.com/luxengine/math` to update your `math` library. It was already fixed before I could file a bug report :smile: .